recipes/default.rb

```diff
@@ -85,19 +88,10 @@
   end
 
   yum_repository 'webtatic' do
-    description (node['librenms']['repo_webtatic']['desc']).to_s
-    baseurl (node['librenms']['repo_webtatic']['url']).to_s
-    gpgcheck false
-    enabled true
-    only_if { node.normal['librenms']['repo_webtatic']['enabled'] = 'true' }
-  end
-
-  yum_repository 'epel' do
-    description (node['librenms']['repo_epel']['desc']).to_s
-    baseurl (node['librenms']['repo_epel']['url']).to_s
+    description node['librenms']['repo_webtatic']['desc']
+    baseurl node['librenms']['repo_webtatic']['url']
     gpgcheck false
```

Comment:
Why don't enforce the gpgp keys? They are present on teh repository: https://repo.webtatic.com/yum/

Reply:
Done!